src/robotFactory.js

```diff
@@ -1,10 +1,94 @@
 'use strict';
 
-class BaseRobot {}
+const ROBOT_BASE_COORD = 0;
```

Comment:
What if we want to change one of the base coordinates?
```suggestion
const ROBOT_BASE_COORDS = {
  x: 0,
  y: 0,
  z: 0
};
```